zerver/tornado/event_queue.py

```diff
@@ -666,7 +666,7 @@ def mark_clients_to_reload(queue_ids: Iterable[str]) -> None:
         reload_web_clients[qid] = True
 
 
-def send_reload_web_client_events(immediate: bool = False, count: Optional[int] = None) -> None:
+def send_reload_web_client_events(immediate: bool = False, count: Optional[int] = None) -> int:
```

Comment:
Maybe worth adding a docstring/comment noting that it returns the number of queues it processed, regardless of whether they accepted the event type... or maybe those are the wrong semantics and it should just loop through `reload_web_clients` until it's actually added events to N queues that accept the event type? 

That would spread load in a slightly more predictable fashion.

Reply:
Yeah, this is a behaviour change that I didn't think about when removing the filtering for `client_name` that I had previously.  Will update to only count one that accept the event.

Reply:
Oh, actually, I already did this right.   The code as currently written only inserts into `reload_web_clients` if they accept the event, so it should only be inserting the event into queues that accept it, and this will be spreading load correctly.